packages/cli/src/commands/apps/settings/update.ts

```diff
@@ -17,6 +17,17 @@ export default class AppSettingsUpdateCommand extends APICommand<typeof AppSetti
 		description: 'the app id',
 	}]
 
+	static examples = [
+		{
+			description: 'update the settings of the app with the given id using the data in "app-settings.json"',
+			command: 'smartthings apps:settings:update 392bcb11-e251-44f3-b58b-17f93015f3aa -i app-settings.json',
+		},
+		{
+			description: 'ask for the ID of an app to update and then update it using the data in "app-settings.json"',
```

Comment:
Sorry to be nit-picky but I wonder if we should change the wording of "ask for the ID of an app" to something like "ask for an app" since we don't really specifically ask them that. We give them a list of apps to choose from and they get to select one. We do list the app id but the name is probably what they're going to go by.

```
$ smartthinngs apps:settings:update
───────────────────────────────────────────────────────────────────────
 #  Display Name                  App Id
───────────────────────────────────────────────────────────────────────
 6  Test App 0001                 e0527c2e-29b1-4252-8d8b-88789cd426c3
 7  Test App 0002                 3e00f9bc-8b10-4263-a885-53dbbef46529
───────────────────────────────────────────────────────────────────────
? Select an app.
```